app/ts-cli/cmd/export.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 Huawei Cloud Computing Technologies Co., Ltd.
```

Comment:
replace copyright year to 2024, you can change author to `openGemini author` like this: `Copyright 2024 The openGemini Authors All rights reserved.`.